static/src/javascripts/projects/common/modules/commercial/adblock-messages.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import config from 'lib/config';
+import detect from 'lib/detect';
+import { local } from 'lib/storage';
+import userFeatures from 'commercial/modules/user-features';
+
+const adblockInUse = () => detect.adblockInUse;
+
+const notMobile = () => detect.getBreakpoint() !== 'mobile';
+
+const isPayingMember = () => userFeatures.isPayingMember();
+
+const visitedMoreThanOnce = () => {
+    const alreadyVisited = local.get('gu.alreadyVisited') || 0;
+
+    return alreadyVisited > 1;
+};
+
+const isAdblockSwitchOn = () => config.switches.adblock;
+
+const noAdblockMsg = () => {
+    if (notMobile()) {
+        if (!visitedMoreThanOnce() || !isAdblockSwitchOn()) {
```

Comment:
We could create a single if block here, rather than having 2 with the same content. eg.

if ( !visitedMoreThanOnce() || !isAdblockSwitchOn() || isPayingMember() ) {
    return adblockInUse();
}